pkg/sriov/list.go

```diff
@@ -0,0 +1,80 @@
+package sriov
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ListNetworkNodeState returns NetworkNodeStates inventory in the given namespace.
+func ListNetworkNodeState(apiClient *clients.Settings,
+	nsname string, options metaV1.ListOptions) ([]*NetworkNodeStateBuilder, error) {
+	glog.V(100).Infof("Listing NetworkNodeStates in the nsname %s with the options %v", nsname, options)
```

Comment:
Change NetworkNodeStates to SriovNetworkNodeStates in all occurrences

Reply:
Changed